### PR TITLE
added complete mamnifest and tests

### PR DESCRIPTION
**Pull request by samMcCarthyBJSS**
Story [AB#8904](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/8904)
Task [AB#9088](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/9088)
         [AB#9089](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/9089)

image

### CODE REVIEW

File: app/pages/summary/complete/ui-tests/general.ui.test.js [hunk 1]
```diff
@@ -0,0 +1,292 @@
...
+    await setState(ClientFunction)('fakeToken', authTokenInSession);
+  }
+  if (setup.getRoute) {
+    mocks();
+  }
+};
+
+const getLocation = ClientFunction(() => document.location.href);
+
+fixture('Order Summary Summary - general')
```
Comment (alibombaye) [inline app/pages/summary/complete/ui-tests/general.ui.test.js:28]: can you update this please. `Order Summary for Complete order - general` perhaps? also can you update the incomplete version of this to something similar.

File: app/pages/summary/incomplete/ui-tests/general.ui.test.js [hunk 2]
```diff
@@ -25,7 +25,7 @@ const pageSetup = async (setup = { withAuth: true, getRoute: true }) => {
 
 const getLocation = ClientFunction(() => document.location.href);
 
-fixture('Order Summary Summary - general')
+fixture('Order Summary for complete order - general')
```
Comment (alibombaye) [inline app/pages/summary/incomplete/ui-tests/general.ui.test.js:28]: `incomplete`

File: app/pages/summary/incomplete/ui-tests/withSavedData.ui.test.js [hunk 3]
```diff
@@ -214,7 +214,7 @@ const pageSetup = async (setup = { withAuth: true, getRoute: true }) => {
   }
 };
 
-fixture('Order Summary - with saved data')
+fixture('Order Summary  for complete order - with saved data')
```
Comment (alibombaye) [inline app/pages/summary/incomplete/ui-tests/withSavedData.ui.test.js:217]: `incomplete`

File: app/pages/summary/contextCreator.test.js [hunk 4]
```diff
@@ -56,6 +63,16 @@ describe('order summary contextCreator', () => {
       expect(context.dateSummaryCreated).toEqual('19 July 2020');
     });
 
+    it('should return the dateCompletedLabel for a completed order', () => {
+      const context = getContext({ orderId: 'order-1', orderData: mockCompletedOrderData });
+      expect(context.dateCompletedLabel).toEqual(completeManifest.dateCompletedLabel);
+    });
+
+    it('should return the mockCompletedOrderData as the current date for a completed order', () => {
```
Comment (alibombaye) [inline app/pages/summary/contextCreator.test.js:71]: think this test description is bit misleading. It's not the currentDate. It'll just be the date that is provided in mockCompletedOrderData

Comment (samMcCarthyBJSS) [reply app/pages/summary/contextCreator.test.js:71]: hm yeah will change that

**Pull request merged**